src/puppetlabs/rbac_client/services/rbac.clj

```diff
@@ -113,9 +113,21 @@
   (init [_ tk-ctx]
     (if-let [rbac-url (get-in-config [:rbac-consumer :api-url])]
       (let [ssl-config (get-in-config [:global :certs])
-            route-limit {:max-connections-per-route 20}
-            certified-client (create-client (merge route-limit ssl-config))
-            uncertified-client (create-client (merge route-limit (select-keys ssl-config [:ssl-ca-cert])))]
+            connection-limits {:max-connections-per-route (get-in-config [:rbac-consumer :max-connections-per-route]
```

Comment:
Similar concept as above, this can be simplified into one map construction with inline defaults of `20`